bolt-sidecar/src/config/mod.rs

```diff
@@ -55,6 +55,9 @@ pub struct Opts {
     /// Max committed gas per slot
     #[clap(long, env = "BOLT_SIDECAR_MAX_COMMITTED_GAS")]
     pub(super) max_committed_gas: Option<NonZero<u64>>,
+    /// Min priority fee to accept for a commitment
+    #[clap(long, env = "BOLT_SIDECAR_MIN_PRIORITY_FEE")]
+    pub(super) min_priority_fee: Option<NonZero<u64>>,
```

Comment:
We should probably add a default for this, like 0.5/1 gwei but let's discuss it a bit

Reply:
Also, isn't `u128` a better type to express priority fees? I'm not sure tho, could you take a look?